packages/@atjson/editor/src/mixins/component.ts

```diff
@@ -0,0 +1,34 @@
+import events from './events';
+
+export default class WebComponent extends events(HTMLElement) {
```

Comment:
Hmm, this looks like a duplicate (and should probably not be in the `mixins` folder)